makefiles/Base.toml

```diff
@@ -8,4 +8,39 @@ CARGO_PROFILE = "release"
 env = { IMAGE_TAG = "${CARGO_MAKE_GIT_HEAD_LAST_COMMIT_HASH}-${CARGO_PROFILE}" }
 
 [env]
-GIT_TAG = { script = ["cat VERSION"] }
+FLUVIO_VERSION = { script = ["cat VERSION"] }
+
+[tasks.build-release.mac]
+args = [
+    "build",
+    "--bin", "fluvio",
+    "--release",
+    "--target=x86_64-apple-darwin",
+    "@@split(CARGO_MAKE_CARGO_BUILD_TEST_FLAGS, )"
+]
+
+[tasks.build-release.linux]
+dependencies = ["setup-musl"]
+args = [
+    "build",
+    "--release",
+    "--bin", "fluvio",
+    "--target=x86_64-unknown-linux-musl",
+    "@@split(CARGO_MAKE_CARGO_BUILD_TEST_FLAGS, )"
+]
+
+[tasks.setup-musl.linux]
+script = '''
+which musl-gcc 2> /dev/null > /dev/null || {
```

Comment:
this is highly specific to ubuntu or OS that has `apt` and doesn't belong here.  it should be done by whatever infrastructure provisioning tools

Reply:
Yeah, I've got mixed feelings about it. This is why the check for `musl-gcc` exists so it doesn't need to be installed. This step is also contingent on it being done with the "production" profile.

Reply:
but that creates dependency to specific infrastructure.  they should move to CI configuration or provisioning tools